cves/kernel/CVE-2018-12232.yml

```
@@ -75,7 +86,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1590215]
```

Can you link this to bugzilla somehow?

---

Thank you for the feedback! I'm going to change it to the code below to add the link. 
```suggestion
bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=1590215"]
```